### PR TITLE
added isString

### PR DESCRIPTION
**Pull request by rubiin**

### CODE REVIEW

**Comment by NamPNQ** [comment]
Lgtm

File: README.md [hunk 1]
```diff
@@ -2351,6 +2352,44 @@ Checks if string ends with the given target string.
...
+ 
+  isString(123);
+  // => false
+  ```
+
+#### Browser Support for `String.prototype.endsWith()`
+
+![Chrome][chrome-image] | ![Edge][edge-image] | ![Firefox][firefox-image] | ![IE][ie-image] | ![Opera][opera-image] | ![Safari][safari-image]
+:-: | :-: | :-: | :-: | :-: | :-: |
+  41.0 ✔  |  ✔ | 17.0 ✔ |  ✖  |  28.0 ✔ |  9.0 ✔ |
```
Comment (ODudek) [inline README.md]: Why did you add `X` in IE?

File: README.md [hunk 2]
```diff
@@ -2351,6 +2352,44 @@ Checks if string ends with the given target string.
...
+
+  ```js
+  // Lodash
+  _.isString('abc');
+  // => true
+ 
+  _.isString(123);
+  // => false
+  
+  _.endsWith('abc', 'b', 2);
```
Comment (ODudek) [inline README.md]: Delete this line :)

Comment (ODudek) [inline README.md]: Delete this line :)

Comment (rubiin) [reply README.md]: which one to be precise

File: README.md (hunk 1 above)
Comment (rubiin) [reply README.md]: let me get that fixed

File: README.md (hunk 2 above)
Comment (NamPNQ) [reply README.md]: Look like you copied from endsWith and forgot remove

File: README.md (hunk 1 above)
Comment (NamPNQ) [reply README.md]: Same above, you forgot update browser support

File: README.md (hunk 2 above)
Comment (rubiin) [reply README.md]: oh right

File: README.md (hunk 1 above)
Comment (rubiin) [reply README.md]: @NamPNQ @ODudek  updated

Comment (rubiin) [reply README.md]: i  will be doing another pr as soon as it gets resolved
